.env.example

```diff
@@ -1,2 +1,4 @@
-COMMITMENT_CONTRACT_ADDRESS=0x00000000000000000000000000000000000000000
 GA_ID=UA-00000000-0
+COMMITMENT_CONTRACT_ADDRESS=0x00000000000000000000000000000000000000000
```

Comment:
add `APP_MODE` env variable which can have 3 different states:
`BEFORE_ANNOUNCEMENT` (do not display exact date) `ANNOUNCED` (display countdown but contracts are not yet deployed) and `ON BLOCKCHAIN` (contracts are there use get everything from blockchain)